packages/data-point/lib/reducer-types/reducer-helpers/reducer-constant/resolve.js

```diff
@@ -0,0 +1,17 @@
+const Promise = require('bluebird')
+
+const utils = require('../../../utils')
+
+/**
+ * @param {Object} manager
+ * @param {Function} resolveReducer
+ * @param {Accumulator} accumulator
+ * @param {ReducerConstant} reducerConstant
+ * @returns {Promise<Accumulator>}
+ */
+function resolve (manager, resolveReducer, accumulator, reducerConstant) {
+  const acc = utils.set(accumulator, 'value', reducerConstant.value)
+  return Promise.resolve(acc)
```

Comment:
executing this code for each constant is too expensive, would rather find a solution at compile time on the ObjectReducer factory

Reply:
This function is not used for constants inside of ReducerObjects.